### PR TITLE
:book: update build badges and triggers in README.md

### PR DESCRIPTION
**Pull request by tuminoid**
Update build badges and test triggers in README.md for release-1.9.

### CODE REVIEW

**Comment by tuminoid** [comment]
/override metal3-centos-e2e-integration-test-main metal3-ubuntu-e2e-integration-test-main

**Comment by tuminoid** [comment]
Were missing some JJB. Fixing that here: https://gerrit.nordix.org/c/infra/cicd/+/22820

**Review by kashifest — COMMENTED** [review]
/approve

**Comment by tuminoid** [comment]
Sorry, this is targeting wrong branch.  It should target 1.9, not main.
/close

File: README.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
+[![CentOS E2E feature 1.9 pivot build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3-periodic-centos-e2e-feature-test-release-1-9-pivoting/&subject=CentOS%20E2E%20feature%201.9%20pivot)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3-periodic-centos-e2e-feature-test-release-1-9-pivoting/)
+[![CentOS E2E feature 1.9 remediation build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3-periodic-centos-e2e-feature-test-release-1-9-remediation/&subject=CentOS%20E2E%20feature%201.9%20remediatoin)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3-periodic-centos-e2e-feature-test-release-1-9-remediation/)
+[![CentOS E2E feature 1.9 other features build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3-periodic-centos-e2e-feature-test-release-1-9-features/&subject=CentOS%20E2E%20feature%201.9%20other%20features)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3-periodic-centos-e2e-feature-test-release-1-9-features/)
```
Comment (adilGhaffarDev) [inline README.md:10]: The status of the main tests should remain unchanged on the main branch. However, we need to update it on the 1.9 branch.

**Pull request closed without merging**